python/ray/tune/test/trial_runner_test.py

```diff
@@ -889,16 +915,6 @@ def train(config, reporter):
                 self.assertLessEqual(len(trial.logdir), 200)
                 trial_executor.stop_trial(trial)
 
-    def testTrialErrorOnStart(self):
-        ray.init()
-        trial_executor = RayTrialExecutor()
-        _global_registry.register(TRAINABLE_CLASS, "asdf", None)
-        trial = Trial("asdf", resources=Resources(1, 0))
-        try:
-            trial_executor.start_trial(trial)
-        except Exception as e:
-            self.assertIn("a class", str(e))
-
```

Comment:
Since `RayTrialExecutor.start_trial` didn't throw, this test didn't actually do anything. I've rewritten and moved it to `ray_trial_executor_test.py`